easybuild/easyblocks/s/suitesparse.py

```diff
@@ -122,7 +124,7 @@ def configure_step(self):
         if cfgvars:
             try:
                 f = open(fp, "a")
-                f.write("# lines below added automatically by EasyBuild")
+                f.write("# lines below added automatically by EasyBuild\n")
                 for (var, val) in cfgvars.items():
                     f.write("%s = %s\n" % (var, val))
                 f.close()
```

Comment:
this whole block could be cleaned up quite a bit with:

```python
cfgtxt = '\n'.join(["%s = %s" % (var, val) for (var, val) in cfgvars.items()])
write_file(fp, cfgtxt, append=True)
```

You can also move it out of the `try`/`except`, since `write_file` already takes care catching `IOError`